src/commonControls/DateControl.ts

```diff
@@ -286,14 +292,21 @@ export interface DateControlPromptProps {
     valueAffirmed?: StringOrList | ((act: ValueConfirmedAct<number>, input: ControlInput) => StringOrList);
 }
 
+export type LastInitiativeState = {
```

Comment:
here and elsewhere...

does it need to be exported?  if yes, it would be good to put it in a namespace e.g. DateControlTypes or similar otherwise I think we will get into name-conflict situations

Reply:
Nope, it does not to be exported, will remove it and change it to an interface!